### DISCUSSION TITLE
7E3 mAb plasma clearance

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1565] [Questions & Problems] [answered]
I am fairly new to large molecule PK modelling and I was trying to see how well PK-Sim's large molecule model works by creating one for a wild type mouse with the 7E3 antibody. The observed data was taken from literature (Shah, 2012).

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/882dada5-b69f-4f19-9020-0ebedb05e09d)

This is what it currently looks like and I'm wondering if it is because I do not have the specific drug properties for 7E3 that made this simulation very off. Does anyone know if there is a database that has the molecular properties of various mAbs? How would I get that clearance that is seen in the observed data that is not reflected in this simulation? 

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/1ad6f0d1-2118-4f2b-be96-1be5f78cbd81)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/640e1d26-dd5f-49fd-a93a-0e2b5663f510)

These are the compound properties and the administration protocol is 8mg/kg. The individual mouse for the simulation is 21g. At this point, I am not even sure which parameters I need to use for the parameter identification. Any help would be greatly appreciated.

**Comment by prvmalik** [comment, marked as answer]
Hi

Uncheck "is small molecule"

Solute radius for a mAb is 5.34 nm (see Niederalt paper)

FcRn affinity Kd seems reasonable

Make sure you select the large molecule model when you have configured your simulation

Paul

**Reply by prvmalik** [reply]
By the way, I recall this dataset has a bit of a shockingly long half life, I think

**Reply by akim2000** [reply]
I swear I unchecked that box when I was making it.. It does have a better clearance now thank you.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/139281283/99d0850f-649b-4abb-b1bb-05c8a0e44f33)

**Comment by Yuri05** [comment]
s. also https://github.com/Open-Systems-Pharmacology/7E3-Model
